src/platform/vita/vita_input.h

```diff
@@ -0,0 +1,24 @@
+#ifndef VITA_INPUT_H
+#define VITA_INPUT_H
+
+#include <SDL2/SDL.h>
+
+int vita_poll_event(SDL_Event *event);
+void vita_handle_analog_sticks(void);
+void vita_handle_virtual_keyboard(void);
+void vita_handle_repeat_keys(void);
+
+#define PAD_UP 8
```

Comment:
```suggestion
#define VITA_PAD_UP 8
```

Reply:
Also, I suggest using an `enum` since those values make sense together.

Reply:
Fixed in 35f42ab00e2b28a49e966a752eb3016a334d53bb